README.md

```diff
@@ -139,13 +139,15 @@ Copy `contrib/.env.sample` as `.env` in the project's root folder and adjust you
 ##### Django settings
 
 * `DEBUG` (_bool_) enable or disable [Django debug mode](https://docs.djangoproject.com/en/1.10/ref/settings/#debug)
+* `GOSS_VERSION` (_str_) [Version for goss tester in docker](https://goss.rocks) 
```

Comment:
Can we use capitalized version for names?  _Version for Goss tester in Docker_ instead of _Version for goss tester in docker_.